PhysicsTools/TensorFlow/bin/test_tfgraph.cc

```diff
@@ -0,0 +1,154 @@
+/*
+ * Test of the TensorFlow graph interface.
+ *
+ * Graph (from bin/data/simplegraph, created by test/creategraph.py):
+ *   Input:
+ *     - op = "input", op index = 0, shape = (batch, 10)
+ *   Output:
+ *     - op name = "output", op index = 0, shape = (batch, 1)
+ *
+ * Usage:
+ *   > test_tfgraph
+ *
+ * Author:
+ *   Marcel Rieger
+ */
+
+#include <stdio.h>
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include <PhysicsTools/TensorFlow/interface/Tensor.h>
+#include <PhysicsTools/TensorFlow/interface/Graph.h>
+
+void test(bool success, const std::string& msg)
+{
+    if (success)
+    {
+        std::cout << "    " << msg << std::endl;
+    }
+    else
+    {
+        throw std::runtime_error("test failed: " + msg);
+    }
+}
+
+int main()
```

Comment:
this file looks like a useful unit test.
Please move it to  PhysicsTools/TensorFlow/test and add it to that directory unit tests.
See e.g. CommonTools/CandUtils/test/testMakeCompositeCandidate.cc and the build file in that same directory as an example